src/components/navBar/index.test.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { render, fireEvent, screen } from '@testing-library/react'
+import NavBar from '.'
+import * as  authHooks from '@/hooks/useAuthenticated';
+
+
+
+describe('Navbar', () => {
+    test('renders the Navbar component', () => {
+        const { getByTestId } = render(<NavBar />);
+        const navbar = getByTestId('navbar');
+        expect(navbar).toBeInTheDocument();
+    });
+
+    test('user whether loggedIn or not', () => {
+        
+        jest.spyOn(authHooks, 'default').mockImplementation(() => (
+            { userData: { userName: "IamYash", 
```

Comment:
test file lives under __tests__/unit folder. Please fix that

Reply:
On it.